dashboard/test/lib/announcements_test.rb

```diff
@@ -72,4 +72,25 @@ class AnnouncementsTest < ActiveSupport::TestCase
     announcement = Announcements.get_announcement_for_page("/courses")
     refute announcement
   end
+
+  test 'returns nil for announcement behind false dcdo flag' do
+    announcement = Announcements.get_announcement_for_page("/dcdo-test")
+    assert_nil announcement
+  end
```

Comment:
This all looks great! My one thought is maybe to explicitly set the DCDO flag here to false at the top of the test the same way you set it to true in the following one? I don't know enough about the behind-the-scenes of how our tests are run and how `datastore_cache` works to know if it would matter either way. Does anyone else have thoughts?